src/test/clitools.rs

```diff
@@ -445,6 +445,127 @@ pub enum Scenario {
     MissingComponentMulti,
 }
 
+impl Scenario {
+    // Creates a mock dist server populated with some test data
+    #[tracing::instrument(level = "trace", skip_all)]
+    fn write(&self, path: &Path) {
```

Comment:
Nit: Naming convention-wise it looks like `fn write(&self, stuff)` is writing something into `self` (see this [example in `std`](https://doc.rust-lang.org/std/io/trait.Write.html#tymethod.write)). Since it's the opposite case here, I propose changing the name to, say, `.write_to(path)`.

Reply:
Good point, I considered the same change but failed to follow through.